src/main/kotlin/org/opensearch/replication/metadata/store/ReplicationMetadataStore.kt

```diff
@@ -265,6 +266,7 @@ class ReplicationMetadataStore constructor(val client: Client, val clusterServic
                 .put(IndexMetadata.INDEX_AUTO_EXPAND_REPLICAS_SETTING.key, "0-1")
                 .put(IndexMetadata.INDEX_PRIORITY_SETTING.key, Int.MAX_VALUE)
                 .put(IndexMetadata.INDEX_HIDDEN_SETTING.key, true)
+                .put(IndexMetadata.INDEX_REPLICATION_TYPE_SETTING.key, ReplicationType.DOCUMENT) // System Indices should use Document Replication strategy
```

Comment:
Can you confirm that `INDEX_REPLICATION_TYPE_SETTING` is not behind feature flag for Segment Replication? 

Just trying to ensure that this works always.

Reply:
Yes INDEX_REPLICATION_TYPE_SETTING  is not behind feature flag anymore. We have removed the feature flag in PR: https://github.com/opensearch-project/OpenSearch/pull/7045, you can see this [here](https://github.com/opensearch-project/OpenSearch/blob/10ab178f54a9d6552d2b48757387071ed9ef8264/server/src/main/java/org/opensearch/common/settings/IndexScopedSettings.java#L104) it is directly added as built in index setting and not gated by any feature flag.